etcd/templates/etcd-svc.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
```

Comment:
Definitely, this should go into ``etcd-petsets.yml`` as it was before. Otherwise you need to create the service first, then the petset.